### PR TITLE
Add decimal list style

### PR DESCRIPTION
**Pull request by deti**
First PR for List 1.1.1. style support for https://trello.com/c/WuwrF8is/8-add-list-style-111

All changes are tested in the Policistat & CKEditor locally.

In the current PR:
* Travis config update
* New list type definition
* CSS styles which do the magic

Next PR would be int ckeditor repo to propagate changes there.

### CODE REVIEW

File: structuredheadings/styles/numbering.css [hunk 1]
```diff
@@ -178,3 +178,20 @@ body {
...
     content: counter(autonumber-5, lower-roman) ". ";
 }
+
+/*
+styles for List: 1. 1. 1.
+inspired by these:
+https://stackoverflow.com/questions/4098195/can-ordered-list-produce-result-that-looks-like-1-1-1-2-1-3-instead-of-just-1
+https://stackoverflow.com/questions/10405945/html-ordered-list-1-1-1-2-nested-counters-and-scope-not-working/10405962
+http://jsfiddle.net/qGCUk/4/
+*/
```
Comment (deti) [inline structuredheadings/styles/numbering.css:188]: I want to keep it at the place here, to save time for googling when something would be broken

File: .travis.yml [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
-sudo: required
+
+services:
+  - xvfb
+
 addons:
-  firefox: "31.0"
-  phantomjs: "1.9.8"
+  firefox: "latest"
+  chrome: stable
```
Comment (jlward) [inline .travis.yml]: Should this one have quotes? Or should the other not have quotes?

File: .travis.yml [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
+
 node_js:
-  - "4.2.1"
+  - 10
```
Comment (jlward) [inline .travis.yml]: Ditto quotes

File: .travis.yml [hunk 4]
```diff
@@ -1,20 +1,23 @@
...
-  - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
+  - 'export DISPLAY=:99.0'
+  - 'sleep 3'
   - npm run-script symlink-plugin-to-ckeditor
   - npm install
+
 env:
   - NPM_CMD=test
-  - NPM_CMD=lint
```
Comment (jlward) [inline .travis.yml:18]: Is there a reason we are no longer linting?

File: structuredheadings/styles/numbering.css [hunk 5]
```diff
@@ -178,3 +178,20 @@ body {
...
+.document .document_body ol.list-decimal-point, .document .document_body ol.list-decimal-point ol {
```
Comment (jlward) [inline structuredheadings/styles/numbering.css]: At no other point this this file are we using selectors that include things outside this repo (`.document` and `.document_body` are constructs in PolicyStat, not the editor. 

Is there a reason we are doing so now? Is it required? Can't we just key on `ol.list-decimal-point`?

File: .travis.yml (hunk 4 above)
Comment (deti) [reply .travis.yml:18]: As far as I see all last runs on the master here are red. And I see a few issues with linting _before_ these changes.

Nobody cared about them for years, and I feel they are no stoppers for bringing it live.

However, I would like to spend a couple days later on enabling linting again with fixing all these issues.

File: structuredheadings/styles/numbering.css (hunk 5 above)
Comment (deti) [reply structuredheadings/styles/numbering.css]: Oh. Right. Copy pasted my older experiments.

**Review by jlward — APPROVED** [review]

**Pull request merged**